.changeset/chilly-dogs-draw.md

```diff
@@ -0,0 +1,8 @@
+---
+"@comet/brevo-admin": major
+"@comet/brevo-api": major
+---
+
+Adds `createBrevoTestContactsPage` for creating a test contacts page, that is indepent from the main list.
+
+Remove `email` and `redirectionUrl` from `brevoContactsPageAttributesConfig`
```

Comment:
Couldn't this be a minor version?

There are no breaking changes as far as I can see. It's a new feature with the test list, but it must not be used as far as I can see.

Reply:
True, I initially thought, that the change in the `brevoContactsPageAttributesConfig` breaks the contacts page and the create/update functions. But that's not true, so minor should be fine 👍

Reply:
Is the change in `EditBrevoContactFormValues` not a breaking change? @raphaelblum @juliawegmayr 
just to be sure, i thought this is breaking since before the email and redirectionUrl had to be included in the `createBrevoContactPage` and now they should not be present there. Or am I wrong here?
 
see changes in `demo/admin/src/common/brevoModuleConfig/brevoContactsPageAttributesConfig.tsx`

```
export type EditBrevoContactFormValues = {
    [key: string]: unknown;
    // email: string; got removed 
    // redirectionUrl: string;  got removed
};
```

Reply:
It is still possible to pass the email and redirectionUrl, or is it not?

Reply:
As long as the test contacts are not used, it is still possible to pass email and redirectionUrl.
I also tested it by using the initial config, like it is now on the main branch.

Reply:
I thought this would result in an error. If not, then there is no problem 👍🏻  sorry for the confusion

Reply:
> As long as the test contacts are not used, it is still possible to pass email and redirectionUrl.

Okay good then it is non-breaking. The target branch can be switched to main as @RainbowBunchie said.